test/drbg_cavs_test.c

```diff
@@ -257,7 +257,7 @@ static int test_cavs_kats(const struct drbg_kat *test[], int i)
 #ifdef FIPS_MODE
     /* FIPS mode doesn't support instantiating without a derivation function */
     if ((td->flags & USE_DF) == 0)
-        return 1;
+        return TEST_skip("no derivation function is untestable in FIPS mode");
```

Comment:
"no derivation function is untestable..."  is a very deep mathematical statement. I'm still not fully aware of all its implications... :wink:

Now seriously: how about giving the following reason?

"instantiating without derivation function is not supported in FIPS mode"

Reply:
> "no derivation function is untestable..." is a very deep mathematical statement.

Sounds epic, gödelian.

Reply:
Changed.

Gödel caused a lot of problems....